tools/specs/add-ons/Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyVaultProvider.nuspec

```diff
@@ -47,19 +47,19 @@ Microsoft.Data.SqlClient.AlwaysEncrypted.AzureKeyStoreProvider.SqlColumnEncrypti
       </group>
     </dependencies>
     <frameworkAssemblies>
-      <frameworkAssembly assemblyName="mscorlib" targetFramework="net46" />
-      <frameworkAssembly assemblyName="System.Data" targetFramework="net46" />
+      <frameworkAssembly assemblyName="mscorlib" targetFramework="net461" />
```

Comment:
This entire file needs to be reverted.

Reply:
Done. I also reverted the package naming.
from AzureKeyVaultProvider\net461\  to AzureKeyVaultProvider\net46\